NuRadioMC/test/Veff/1e18eV/T03check_output.py

```diff
@@ -41,12 +39,9 @@ def calculate_veff(fin):
 
 print('fraction of triggered events = {:.0f}/{:.0f} = {:.3f} -> {:.6g} km^3 sr'.format(n_triggered, n_events, n_triggered / n_events, Veff / units.km ** 3))
 
-delta = (Veff / units.km ** 3 - Veff_mean) / Veff_sigma
 rdelta = (Veff / units.km ** 3 - Veff_mean) / Veff_mean
```

Comment:
Why did you change that? This makes the test much less stringent.

Reply:
No, it was actually not used to decide the outcome of the test. It was only used to report the deviation in sigmas which since `Veff_sigma = 0.0001` it reported crazy high numbers which did not help in understanding what happens.

Reply:
@cg-laser ?

Reply:
I really would like to get this merged asap. So please @cg-laser comment. My changes do not change what the test does. They only change what numbers are printed into the terminal. But I think these numbers were anyway very misleading.

Reply:
@cg-laser can you please resolve the request for change or comment?

Reply:
@cg-laser apologies for the change and my pushy behaviour. Indeed I changed the test condition. I changed that back to 3 sigma (sigma is a very small number). However, we are not reporting the deviation in sigma anymore but in %. This was, imo, very confusing and the reason I made the change in the first place. 

This being said, I noticed that the to `T03_check*py` files are basically the same besides 1 hard-coded float and one string (trigger name). I removed one file and allowed both tests to use one file.